SingularityUI/app/components/dashboard/DashboardPage.jsx

```diff
@@ -0,0 +1,199 @@
+import React, { Component, PropTypes } from 'react';
+import { connect } from 'react-redux';
+import { Row, Col } from 'react-bootstrap';
+
+import Utils from '../../utils';
+
+import * as StarredActions from '../../actions/ui/starred';
+import * as RequestActions from '../../actions/api/request';
+import * as RequestsSelectors from '../../selectors/requests';
+
+import UITable from '../common/table/UITable';
+import { Starred, RequestId, Type, LastDeploy, DeployUser, State, Actions } from '../requests/Columns';
+
+import RequestCounts from './RequestCounts';
+import RequestCount from './RequestCount';
+
+class DashboardPage extends Component {
```

Comment:
I would recommend breaking this up into separate components and many doing the container / dumb component pattern here.